wrong folder kickstart

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->


**What happened**:
```
$ ./local-up-karmada.sh 
Creating cluster "karmada-host" ...
 ✓ Ensuring node image (kindest/node:v1.21.1) 🖼 
 ✓ Preparing nodes 📦  
 ✓ Writing configuration 📜 
 ✓ Starting control-plane 🕹️ 
 ✓ Installing CNI 🔌 
 ✓ Installing StorageClass 💾 
 ✓ Waiting ≤ 2m0s for control-plane = Ready ⏳ 
 • Ready after 30s 💚
Set kubectl context to "kind-karmada-host"
You can now use your cluster with:

kubectl cluster-info --context kind-karmada-host --kubeconfig /root/.kube/karmada.config

Have a nice day! 👋
Context "kind-karmada-host" renamed to "karmada-host".
Cluster "kind-karmada-host" set.
make: *** No rule to make target 'images'.  Stop.
$ docker ps
CONTAINER ID   IMAGE                                        COMMAND                  CREATED              STATUS              PORTS                        NAMES
f74e81970c4e   kindest/node:v1.21.1                         "/usr/local/bin/entr…"   About a minute ago   Up About a minute   127.0.0.1:46743->6443/tcp    karmada-host-control-plane
```
**What you expected to happen**:
either choose to continue on a different context or stop earlier!

**How to reproduce it (as minimally and precisely as possible)**:
```
$ git clone ; cd karmada/hack ; ./local-up-karmada.sh 
```

--- a/hack/local-up-karmada.sh
+++ b/hack/local-up-karmada.sh
@@ -17,6 +17,10 @@ SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 # The KUBECONFIG path for the 'host cluster'.
 HOST_CLUSTER_KUBECONFIG="${KUBECONFIG_PATH}/karmada.config"
 
+# Make sure go exists
+source "${SCRIPT_ROOT}"/hack/util.sh
+util::cmd_must_exist "go"
+
 # Make sure KUBECONFIG path exists.
 if [ ! -d "$KUBECONFIG_PATH" ]; then
   mkdir -p "$KUBECONFIG_PATH"
@@ -28,7 +32,7 @@ fi
 # make controller-manager image
 export VERSION="latest"
 export REGISTRY="swr.ap-southeast-1.myhuaweicloud.com/karmada"
-make images
+make images --directory="${SCRIPT_ROOT}"
 
 # load controller-manager image
 kind load docker-image "${REGISTRY}/karmada-controller-manager:${VERSION}" --name="${HOST_CLUSTER_NAME}"

